package.json

```diff
@@ -17,7 +18,7 @@
   },
   "scripts": {
     "release": "changeset publish",
-    "test": "eslint .",
+    "test": "eslint . && eslint -c base.js --no-eslintrc .",
```

Comment:
```suggestion
    "test": "eslint . && eslint --config base.js --no-eslintrc .",
```

Reply:
I've removed .eslintrc.js and made the test command explicit about which file it's using to test.